components/calendar/src/lib.rs

```diff
@@ -88,6 +88,7 @@
     )
 )]
 #![warn(missing_docs)]
+#![cfg_attr(docsrs, feature(doc_cfg, doc_auto_cfg))]
```

Comment:
Why only docsrs?

Reply:
It needs nightly and that's the cfg that docs.rs uses. No point introducing a second one.